CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing
```

Comment:
you can add also the branch name based on:
- for new feature: `feature/<task-name>`
- for bug: `bugfix/<task-name>`